### ISSUE TITLE
error installing topojson using npm in windows 8

### ISSUE THREAD

**Issue by dedee-14** [mbostock/topojson#216]
Hallo, I'm a newbie for topojson, I just installed the newest version node.js but I couldnt finish install topojson with command below:
npm install -g topojson

Here's my error  message 

Microsoft Windows [Version 6.3.9600]
(c) 2013 Microsoft Corporation. All rights reserved.

C:\Users\dian anggraeni>node -v
v0.10.34

C:\Users\dian anggraeni>npm -v
1.4.28

C:\Users\dian anggraeni>npm install -g topojson

> contextify@0.1.9 install C:\Users\dian anggraeni\AppData\Roaming\npm\node_modu
les\topojson\node_modules\d3\node_modules\jsdom\node_modules\contextify
> node-gyp rebuild

C:\Users\dian anggraeni\AppData\Roaming\npm\node_modules\topojson\node_modules\d
3\node_modules\jsdom\node_modules\contextify>node "C:\Program Files\nodejs\node_
modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebui
ld
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable "python", you can set the PYT
HON env variable.
gyp ERR! stack     at failNoPython (C:\Program Files\nodejs\node_modules\npm\nod
e_modules\node-gyp\lib\configure.js:103:14)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node
-gyp\lib\configure.js:64:11
gyp ERR! stack     at Object.oncomplete (evalmachine.<anonymous>:108:15)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modu
les\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\dian anggraeni\AppData\Roaming\npm\node_modules\topojson\n
ode_modules\d3\node_modules\jsdom\node_modules\contextify
gyp ERR! node -v v0.10.34
gyp ERR! node-gyp -v v1.0.1
gyp ERR! not ok
npm ERR! contextify@0.1.9 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the contextify@0.1.9 install script.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nod
ejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Users\dian anggraeni
npm ERR! node -v v0.10.34
npm ERR! npm -v 1.4.28
npm ERR! code ELIFECYCLE
npm ERR! not ok code 0

Could you tell me what's wrong and help me to solve the problem? Thanks.

**Comment by tdhsmith** [comment]
The [contextify package](https://github.com/brianmcd/contextify) (which this one depends on) has some manual installation steps on Windows. They provide a guide [here](https://github.com/brianmcd/contextify/wiki/Windows-Installation-Guide), which admittedly is a little hard to follow. I found [these instructions on the jsdom project page](https://github.com/tmpvar/jsdom#windows) to be the most helpful for Windows 7, but I cannot make any guarantees for Win8. 

FYI, pay close attention to where in the process the error occurs. npm doesn't give super straightforward errors, but notice halfway down:
```
npm ERR! Failed at the contextify@0.1.9 install script.
npm ERR! This is most likely a problem with the contextify package,
```
This isn't the package you're installing (`topojson`) so it must be a problem with a dependency. Usually the best place to start looking for answers is going to be their page, even if you have no idea what the package is or does (which will happen a lot :stuck_out_tongue:). Cheers!

**Comment by dedee-14** [comment]
thank you so much for your informative answer. Takes too long for me to check all packages and dependencies in windows so I decided installing topojson in Ubuntu using virtual machine, which I found faster and easier. Luckily, it works perfectly right in time. cheers :)

**Comment by kwoxer** [comment]
Mhh I could also use a Linux for that. But why. So I have studied out how to install **topoJSON** on a **Windows**:
http://w3facility.org/question/topojson-build-fails-in-windows/
http://gis.stackexchange.com/questions/45138/convert-geojson-to-topojson
http://stackoverflow.com/questions/16628638/how-to-run-topojson
But I still have issues. First of all the error log:

```
npm http GET https://registry.npmjs.org/hawk/-/hawk-1.1.1.tgz
npm http 200 https://registry.npmjs.org/tough-cookie

> contextify@0.1.11 install C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\t
opojson\node_modules\d3\node_modules\jsdom\node_modules\contextify
> node-gyp rebuild

npm http 200 https://registry.npmjs.org/hawk/-/hawk-1.1.1.tgz
npm http GET https://registry.npmjs.org/domutils
npm http GET https://registry.npmjs.org/readable-stream
npm http GET https://registry.npmjs.org/entities

npmC:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_modules\d3\n
ode_modules\jsdom\node_modules\contextify>node "C:\Program Files (x86)\nodejs\no
de_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" re
build
http GET https://registry.npmjs.org/domelementtype
npm http GET https://registry.npmjs.org/domhandler
npm http 200 https://registry.npmjs.org/domhandler
npm http 200 https://registry.npmjs.org/domelementtype
npm http 200 https://registry.npmjs.org/domutils
npm http 200 https://registry.npmjs.org/entities
npm http 200 https://registry.npmjs.org/readable-stream
npm http GET https://registry.npmjs.org/readable-stream/-/readable-stream-1.1.13
-1.tgz
npm http 200 https://registry.npmjs.org/readable-stream/-/readable-stream-1.1.13
-1.tgz
gyp ERR! configure error
gyp ERR! stack Error: Command failed: CreateProcessW: Zugriff verweigert
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:540:15)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:99:17)
gyp ERR! stack     at maybeClose (child_process.js:638:16)
gyp ERR! stack     at Socket.ChildProcess.spawn.stdin (child_process.js:815:11)
gyp ERR! stack     at Socket.EventEmitter.emit (events.js:96:17)
gyp ERR! stack     at Socket._destroy.destroyed (net.js:358:10)
gyp ERR! stack     at process.startup.processNextTick.process._tickCallback (nod
e.js:245:9)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\nod
e_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_mo
dules\d3\node_modules\jsdom\node_modules\contextify
gyp ERR! node -v v0.8.22
gyp ERR! node-gyp -v v0.8.5
gyp ERR! not ok
npm ERR! error rolling back Error: ENOTEMPTY, rmdir 'C:\Users\Mr. Kwox\AppData\R
oaming\npm\node_modules\topojson\node_modules\d3\node_modules\jsdom\node_modules
\cssstyle\lib\properties'
npm ERR! error rolling back  jsdom@1.0.0 { [Error: ENOTEMPTY, rmdir 'C:\Users\Mr
. Kwox\AppData\Roaming\npm\node_modules\topojson\node_modules\d3\node_modules\js
dom\node_modules\cssstyle\lib\properties']
npm ERR! error rolling back   errno: 53,
npm ERR! error rolling back   code: 'ENOTEMPTY',
npm ERR! error rolling back   path: 'C:\\Users\\Mr. Kwox\\AppData\\Roaming\\npm\
\node_modules\\topojson\\node_modules\\d3\\node_modules\\jsdom\\node_modules\\cs
sstyle\\lib\\properties' }
npm ERR! EEXIST, mkdir 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topoj
son\node_modules\d3\node_modules\jsdom\node_modules\cssstyle\lib\properties'
File exists: C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_mo
dules\d3\node_modules\jsdom\node_modules\cssstyle\lib\properties
Move it away, and try again.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\cssstyle\lib\properties
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\cssstyle\lib\properties\webkit
BorderStart.js
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code EEXIST
npm ERR! errno 47
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:171:23
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\mkdirp\index.js:37:53
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR! Error: EPERM, open 'C:\Program Files (x86)\nodejs\npm-debug.log'
npm ERR!  { [Error: EPERM, open 'C:\Program Files (x86)\nodejs\npm-debug.log']
npm ERR!   errno: 50,
npm ERR!   code: 'EPERM',
npm ERR!   path: 'C:\\Program Files (x86)\\nodejs\\npm-debug.log' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Program Files (x86)\nodejs\npm-debug.log
npm ERR! code EPERM
npm ERR! errno 50
npm ERR! stack Error: EPERM, open 'C:\Program Files (x86)\nodejs\npm-debug.log'
npm ERR! contextify@0.1.11 install: `node-gyp rebuild`
npm ERR! `cmd "/c" "node-gyp rebuild"` failed with 1
npm ERR!
npm ERR! Failed at the contextify@0.1.11 install script.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! code ELIFECYCLE
npm http GET https://registry.npmjs.org/core-util-is
npm http GET https://registry.npmjs.org/isarray/0.0.1
npm http GET https://registry.npmjs.org/string_decoder
npm http GET https://registry.npmjs.org/inherits
npm http 200 https://registry.npmjs.org/isarray/0.0.1
npm http 200 https://registry.npmjs.org/string_decoder
npm http 200 https://registry.npmjs.org/core-util-is
npm http 200 https://registry.npmjs.org/inherits
npm http GET https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz
npm http 200 https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz
npm http GET https://registry.npmjs.org/delayed-stream/0.0.5
npm http GET https://registry.npmjs.org/async
npm http 200 https://registry.npmjs.org/delayed-stream/0.0.5
npm http GET https://registry.npmjs.org/assert-plus
npm http GET https://registry.npmjs.org/asn1/0.1.11
npm http GET https://registry.npmjs.org/ctype/0.5.3
npm http GET https://registry.npmjs.org/delayed-stream/-/delayed-stream-0.0.5.tg
z
npm http 200 https://registry.npmjs.org/async
npm http 200 https://registry.npmjs.org/assert-plus
npm http 200 https://registry.npmjs.org/asn1/0.1.11
npm http 200 https://registry.npmjs.org/ctype/0.5.3
npm http 200 https://registry.npmjs.org/delayed-stream/-/delayed-stream-0.0.5.tg
z
npm http GET https://registry.npmjs.org/ctype/-/ctype-0.5.3.tgz
npm http GET https://registry.npmjs.org/asn1/-/asn1-0.1.11.tgz
npm ERR! error rolling back Error: ENOTEMPTY, rmdir 'C:\Users\Mr. Kwox\AppData\R
oaming\npm\node_modules\topojson\node_modules\d3\node_modules\jsdom\node_modules
\request\node_modules\hawk\test'
npm ERR! error rolling back  request@2.51.0 { [Error: ENOTEMPTY, rmdir 'C:\Users
\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_modules\d3\node_modules
\jsdom\node_modules\request\node_modules\hawk\test']
npm ERR! error rolling back   errno: 53,
npm ERR! error rolling back   code: 'ENOTEMPTY',
npm ERR! error rolling back   path: 'C:\\Users\\Mr. Kwox\\AppData\\Roaming\\npm\
\node_modules\\topojson\\node_modules\\d3\\node_modules\\jsdom\\node_modules\\re
quest\\node_modules\\hawk\\test' }
npm http 200 https://registry.npmjs.org/asn1/-/asn1-0.1.11.tgz
npm http 200 https://registry.npmjs.org/ctype/-/ctype-0.5.3.tgz
npm ERR! Error: ENOENT, lstat 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_module
s\topojson\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\
form-data\node_modules\mime-types\README.md'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\request\node_modules\form-data\node_mo
dules\mime-types\README.md
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\form-data
\node_modules\mime-types\README.md
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR! Error: ENOENT, lstat 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_module
s\topojson\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\
hawk\test\browser.js'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\request\node_modules\hawk\test\browser
.js
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\hawk\test
\browser.js
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR! Error: ENOENT, lstat 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_module
s\topojson\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\
bl\node_modules\readable-stream\lib\_stream_readable.js'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\request\node_modules\bl\node_modules\r
eadable-stream\lib\_stream_readable.js
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\bl\node_m
odules\readable-stream\lib\_stream_readable.js
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR! Error: ENOENT, lstat 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_module
s\topojson\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\
form-data\node_modules\async\README.md'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\request\node_modules\form-data\node_mo
dules\async\README.md
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\form-data
\node_modules\async\README.md
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR! Error: ENOENT, lstat 'C:\Users\Mr. Kwox\AppData\Roaming\npm\node_module
s\topojson\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\
tough-cookie\lib\pubsuffix.js'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files (x86)\\nodejs\\\\node.exe" "C:\\Program File
s (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "topojson"
npm ERR! cwd C:\Program Files (x86)\nodejs
npm ERR! node -v v0.8.22
npm ERR! npm -v 1.2.14
npm ERR! path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojson\node_m
odules\d3\node_modules\jsdom\node_modules\request\node_modules\tough-cookie\lib\
pubsuffix.js
npm ERR! fstream_path C:\Users\Mr. Kwox\AppData\Roaming\npm\node_modules\topojso
n\node_modules\d3\node_modules\jsdom\node_modules\request\node_modules\tough-coo
kie\lib\pubsuffix.js
npm ERR! fstream_type File
npm ERR! fstream_class FileWriter
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! fstream_stack C:\Program Files (x86)\nodejs\node_modules\npm\node_modul
es\fstream\lib\writer.js:284:26
npm ERR! fstream_stack Object.oncomplete (fs.js:297:15)
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Program Files (x86)\nodejs\npm-debug.log
npm ERR! not ok code 0
```

**Comment by kwoxer** [comment]
Installing contextify with npm also failed. Now trying https://github.com/TooTallNate/node-gyp/wiki/Visual-Studio-2010-Setup and coming back with more infos then.

**Comment by kwoxer** [comment]
Alright got it working finally... puh. That's super tricky under Windows.

Gonna write a tutorial about that soon and paste link here when finished.

And btw topoJSON is great. Thanks for the awsome work guys.

**Comment by tdhsmith** [comment]
Haha awesome news, because I was just about to retrace my steps. :stuck_out_tongue:  I definitely never encountered any errors in German though, so congrats on overcoming that!

**Comment by mbostock** [comment]
FWIW, I have plans to remove the JSDOM dependency from D3, which will remove the contextify dependency and thus make it easier to install topojson. (I may still use JSDOM for D3 testing, but hopefully can do that in a way that doesn’t require people to install it when D3 is merely an indirect dependency.)

**Comment by kwoxer** [comment]
http://www.kwoxer.de/2015/01/26/topojson-contextify-node-gym-unter-windows-installieren-tutorial/

Hope it helps =)

**Comment by Ciwan1859** [comment]
@kwoxer are you aware of such a guide in English? This is soooo difficult to install! I am on Windows 8 too :( :(

**Comment by kwoxer** [comment]
Well just ask. You can also use a translator on my side. Don't the pictures help you? Just ask when hasving issues my friend. =)

**Comment by Ciwan1859** [comment]
When I translate via Chrome, it says that I must uninstall Visual C++ 2010, is that right? And wouldn't this break other .NET related stuff that I do in Visual Studio?

**Comment by kwoxer** [comment]
Well I also tested it with VS 2013, but I uninstalled 2010 before that and installed 2010 Express. I just did what another tutorial told me. But I personally think that you can leave the installed 2010 version. Just do the SDK stuff and so on and skin the unstalling 2010 and reinstalling 2010 thing.

Try it and report if this works please.

**Comment by mbostock** [comment]
D3 3.5.4 no longer has JSDOM as a dependency, so this should make it much easier to install TopoJSON on Windows. (It’s still a development dependency for D3 tests, but D3’s development dependencies aren’t needed to install TopoJSON.)
